src/source/Metrics/Metrics.c

```diff
@@ -7,11 +7,48 @@
 STATUS getIceCandidatePairStats(PRtcPeerConnection pRtcPeerConnection, PRtcIceCandidatePairStats pRtcIceCandidatePairStats)
 {
     STATUS retStatus = STATUS_SUCCESS;
-    PKvsPeerConnection pKvsPeerConnection = (PKvsPeerConnection) pRtcPeerConnection;
-    UNUSED_PARAM(pKvsPeerConnection);
+    BOOL locked = FALSE;
+    PIceAgent pIceAgent = ((PKvsPeerConnection) pRtcPeerConnection)->pIceAgent;
```

Comment:
pRtcPeerConnection might be NULL. You can define the pIceAgent = NULL and set it after the checks

Reply:
Done

Reply:
Done